infra/algorithm/key-generator/cosid/pom.xml

```diff
@@ -38,7 +38,7 @@
         </dependency>
         <dependency>
             <groupId>org.apache.shardingsphere</groupId>
-            <artifactId>shardingsphere-infra-algorithm-key-generator-core</artifactId>
+            <artifactId>shardingsphere-infra-key-generator-core</artifactId>
```

Comment:
Can you check the latest code? I found `shardingsphere-infra-algorithm-key-generator-core` is right artifactId. https://github.com/apache/shardingsphere/blob/05f445463cd5c3f5e774a5eba26100d6361b9e88/infra/algorithm/key-generator/core/pom.xml#L26